python/xorbits/deploy/kubernetes/config.yml

```diff
@@ -0,0 +1,7 @@
+"@inherits": ../oscar/base_config.yml
+cluster:
+  backend: k8s
+storage:
+  backends: [shared_memory]
+  plasma:
```

Comment:
No need to specify plasma params if choose shared_memory.

Reply:
Fix